src/wagtailmath/templatetags/wagtailmath.py

```diff
@@ -0,0 +1,11 @@
+from django import template
+
+from wagtailmath.widgets import MATHJAX_VERSION
+
+
+register = template.Library()
+
+
+@register.simple_tag
+def mathjax(config="TeX-MML-AM_CHTML"):
+    return f"https://cdnjs.cloudflare.com/ajax/libs/mathjax/{MATHJAX_VERSION}/MathJax.js?config={config}"
```

Comment:
**Thought (non-blocking):** I assume this means this library won't be compatible with strict Content Security Policies because we load JS from an external source. Would it be possible to remove this external dependency?

Reply:
this one is a bit more tricky as depending on the passed config, this will download extra JS.

One to perhaps fix in #13